setup.py

```diff
@@ -30,8 +30,10 @@
     'scripts': glob.glob('scripts/*'),
     'version': version.version,
     'include_package_data': True,
-    'install_requires': ['numpy>=1.10', 'astropy>=1.2', 'scipy', 'mpi4py',
-                         'pyyaml', 'six>=1.10', 'pyuvdata'],
+    'test_suite': 'nose.collector'
+    'test_requires': ['nose', 'h5py'],
+    'install_requires': ['numpy>=1.14', 'astropy>=2.0', 'scipy', 'mpi4py>1.3.1',
+                         'pyyaml', 'six>=1.10', 'pyuvdata>=1.3'],
```

Comment:
I think the mpi4py version requirement should be higher than this, but I don't know what it should be.
I think they pyuvdata version requirement may also need to be higher, but I'm not sure what it should be.

Reply:
`mpi4py` is a tricky requirement. The conda package installs a particular version of MPI with it, and wraps those functions, but it can be built against any MPI installation. That's what's currently done in Travis in order to support MPI-3 functions. I'd like to stick to MPI-3 so we can use shared memory in the future, but I'm not sure what the minimum mpi4py requirement is to support that (I think it's had wrappers for MPI-3 functions for some time...). At the very least, we need a version of mpi4py that's Python-3 compatible.

The newest pyuvdata function I can think of that's required is Nick's new frequency interpolation for healpix beams. So whatever version was in place on March 30th should be the minimum requirement.

Reply:
numpy should be >=1.15